srcpkgs/reddio/patches/change_share_path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/reddio b/reddio
+index dd65718..bc78105 100755
+--- a/reddio
++++ b/reddio
+@@ -197,7 +197,7 @@ main()
+ 	fi
+ 
+ 	# Library path environment variable has precedence over config
+-	lib_dir=${REDDIO_LIB:-${lib_dir:-/usr/local/share/reddio}}
++	lib_dir=${REDDIO_LIB:-${lib_dir:-/usr/share/reddio}}
```

Comment:
can we use `build_style=gnu-makefile` and set `make_install_args="LIBDIR=/usr/share/reddio"` instead?